modules/opp-architecture-architecture.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * architecture/opp-architecture.adoc
+
+:_module-type: CONCEPT
+[id="opp-architecture-architecture_{context}"]
+= Architecture
+
+{product-title} builds on the capabilities of {ocp} with advanced multi-cluster security features, complete management capabilities, integrated data management, and a global container Registry.
```

Comment:
Please confirm that "Registry" be capitalized here.

Reply:
Verified on the portal and corrected.